pages/history.tsx

```diff
@@ -9,27 +9,79 @@ import TransactionsContainer from 'sections/history/TransactionsContainer';
 import StatsSection from 'components/StatsSection';
 
 import StatBox from 'components/StatBox';
-import useSynthetixQueries from '@synthetixio/queries';
+import { issuance, SynthetixQueryContext } from '@synthetixio/queries';
 import { useRecoilValue } from 'recoil';
 import { walletAddressState } from 'store/wallet';
+import { StakingTransactionType } from 'sections/history/types';
+import _ from 'lodash';
+import React from 'react';
```

Comment:
Can you import `{Context}` on line one instead?

Reply:
https://github.com/Synthetixio/staking/pull/769/commits/0a9be4a938eb698efd5a4b16244d182fdacbbbb5